releasenotes/notes/fix-dd-tags-parser-81ba37fbf2c6e879.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixed support for URLs in the ``DD_TAGS`` environment variable.
```

Comment:
Let's outline the cases we know where a currently working string will no longer work to help those that might run into issues